### PR TITLE
Add a couple of ~recent news items

### PR DESCRIPTION
**Pull request by joshuagl**
Add news entries for:
* New spec format and auto-publishing
* TUF primitives powering Sigstore's trust root

### CODE REVIEW

File: content/news.md [hunk 1]
```diff
@@ -4,13 +4,31 @@ title: News
 
 > The [press](/press) page contains a full listing of news coverage.
 
+**June 16, 2021**
+
+The Sigstore community live-streams [key generation and signing ceremony
```
Comment (mnm678) [inline content/news.md:9]:
```suggestion
The Sigstore community live-streamed a [key generation and signing ceremony
```

**Review by mnm678 — APPROVED** [review]
I commented on one sentence that felt a bit awkward, but otherwise LGTM and thanks for the updates!

**Comment by joshuagl** [comment]
Thanks for the review Marina. I applied your suggestion, but I don't have permissions to merge PRs here.

**Comment by mnm678** [comment]
> Thanks for the review Marina. I applied your suggestion, but I don't have permissions to merge PRs here.

I don't either, maybe @jhdalek55 or @JustinCappos can take a look.

**Comment by jhdalek55** [comment]
I'd be happy to take a look at the text, but I don't have permissions on
this site as well and I'm not sure who does. It makes it hard to keep the
site updated.

Lois

**Comment by mnm678** [comment]
cc @trishankatdatadog

**Review by trishankatdatadog — APPROVED** [review]

**Pull request merged**